application/gqlgen/mutations.go

```diff
@@ -162,3 +162,51 @@ func (r *mutationResolver) SetThreadLastViewedAt(ctx context.Context, input SetT
 
 	return &thread, nil
 }
+
+// CreateOrganizationTrust establishes a OrganizationTrust between two organizations
+func (r *mutationResolver) CreateOrganizationTrust(ctx context.Context, input CreateOrganizationTrustInput) (*models.Organization, error) {
+	cUser := models.GetCurrentUserFromGqlContext(ctx)
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+	}
+
+	var organization models.Organization
+	if err := organization.FindByUUID(input.PrimaryID); err != nil {
+		return nil, reportError(ctx, err, "CreateOrganizationTrust.FindPrimaryOrganization", extras)
+	}
+
+	if !cUser.CanCreateOrganizationTrust() {
+		err := errors.New("insufficient permissions")
+		return nil, reportError(ctx, err, "CreateOrganizationTrust.Unauthorized", extras)
+	}
+
+	if err := organization.CreateTrust(input.SecondaryID); err != nil {
+		return nil, reportError(ctx, err, "CreateOrganizationTrust", extras)
+	}
+
+	return &organization, nil
+}
+
+// RemoveOrganizationTrust removes a OrganizationTrust between two organizations
+func (r *mutationResolver) RemoveOrganizationTrust(ctx context.Context, input RemoveOrganizationTrustInput) (*models.Organization, error) {
+	cUser := models.GetCurrentUserFromGqlContext(ctx)
+	extras := map[string]interface{}{
+		"user": cUser.UUID,
+	}
+
+	var organization models.Organization
+	if err := organization.FindByUUID(input.PrimaryID); err != nil {
+		return nil, reportError(ctx, err, "RemoveOrganizationTrust.FindPrimaryOrganization", extras)
+	}
+
+	if !cUser.CanCreateOrganizationTrust() {
```

Comment:
I'm thinking simple Org Admins should be able to remove an OrgTrust for their own org. @fillup, what do you think?

Reply:
Sure, seems fair. But I'm okay to leave it this way for now too since we wont need that ability for a while.